modules/documentscanadd.php

```diff
@@ -23,6 +23,7 @@
  *
  *  $Id$
  */
+$allow_files = ConfigHelper::getConfig('phpui.allowed_documents_scan_filetypes', 'application/pdf, images/jpeg, images/jpg');
```

Comment:
Myślę, że tu byłu niepoprawne typy mime - poprawiłem to już. Dodałem również `image/png`, bo chyba obsługujemy go?

Reply:
A faktycznie, dzięki.

Nie znam żadnego skanera, który skanuje do png - wydaje mi się, więc to więc zbyteczne w domyślnym ustawieniu.
Docelowo ja i tak planuję `phpui.allowed_document_scan_mime_types` = `application/pdf`.